### PR TITLE
[Opinions needed]Add linting and formating config for code and style consistency

### PR DESCRIPTION
**Pull request by abu-hasib**

### CODE REVIEW

File: cubeseed_login/.husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+cd cubeseed_login
+# npx pretty-quick --staged
+# npm run pre-commit
```
Comment (sgarciahelguera) [inline cubeseed_login/.husky/pre-commit:6]: Shouldn't you uncomment those? 
What does the husky.sh do?
Where is it?

Comment (abu-hasib) [reply cubeseed_login/.husky/pre-commit:6]: I will, [Husky](https://typicode.github.io/husky/) prevent commits till all the linting issues are fixed.

Comment (sgarciahelguera) [reply cubeseed_login/.husky/pre-commit:6]: But where is the husky.sh file? and what's in it?

Comment (abu-hasib) [reply cubeseed_login/.husky/pre-commit:6]: It is created when you run npm run prepare

Comment (abu-hasib) [reply cubeseed_login/.husky/pre-commit:6]: closes #24

**Comment by abu-hasib** [comment]
@sgarciahelguera can you approve this please? I already made a separate PR to fix the build issue

**Comment by abu-hasib** [comment]
OK @MimaoQiuQiu <https://github.com/MimaoQiuQiu> please approve this

**Review by MimaoQiuQiu — APPROVED** [review]

**Pull request merged**